0000-notes-on-oer-encoding/0000-notes-on-oer-encoding.md

```diff
@@ -0,0 +1,285 @@
+---
+title: Notes on OER Encoding
+draft: 1
+---
+
+# Introduction
+
+The Interledger stack consists of a collection of different protocols. In order to keep some consistency between these separate, but related protocols, the Interledger team has decided to use some common choices for encoding formats across at least those protocols we control.
```

Comment:
I would describe the "Interledger community" as made up of potentially several "teams"?